### PR TITLE
Update DCP_change_log.md

### PR DESCRIPTION
**Pull request by LouisDJ-SL**

### CODE REVIEW

File: release-notes/Cloud/DCP_change_log.md [hunk 1]
```diff
@@ -9,6 +9,14 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### 9 December 2024 - Enhancement - Home/Admin/Catalog/Marketplace/Sharing - Native link element for app titles [ID 51524]
+
+The app title of the following apps: Admin, Catalog, Home, Marketplace & Sharing now use native link element, allowing users to open the apps in a new tab using middle-click or right-click options.
+
+### 9 December 2024 - Enhancement - Home - Updated terms of service and conditions for DaaS deployments [ID 41651]
+
+The link to the correct terms of service and conditions for DaaS deployments has been updated.
```
Comment (MariekeGO) [inline release-notes/Cloud/DCP_change_log.md:18]: Do you mean the link was incorrect previously? In that case, this should probably be indicated as a fix. Or if it wasn't incorrect, we should remove the word "correct", because that will otherwise give the wrong impression.

File: release-notes/Cloud/DCP_change_log.md [hunk 2]
```diff
@@ -9,6 +9,14 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### 9 December 2024 - Enhancement - Home/Admin/Catalog/Marketplace/Sharing - Native link element for app titles [ID 51524]
+
+The app title of the following apps: Admin, Catalog, Home, Marketplace & Sharing now use native link element, allowing users to open the apps in a new tab using middle-click or right-click options.
```
Comment (MariekeGO) [inline release-notes/Cloud/DCP_change_log.md:14]: Could you rephrase this so it's more from the perspective of the user? "Native link element" might not mean much to them. Note also that the ID for this release note doesn't seem to exist, so possible there's a mistake there.

Comment (LouisDJ-SL) [reply release-notes/Cloud/DCP_change_log.md:14]: Oh, my bad, first number should be a 4

File: release-notes/Cloud/DCP_change_log.md (hunk 1 above)
Comment (LouisDJ-SL) [reply release-notes/Cloud/DCP_change_log.md]: Removed the word "correct" since the link was good but the (toc) page was updated.

File: release-notes/Cloud/DCP_change_log.md [hunk 3]
```diff
@@ -9,6 +9,14 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### 9 December 2024 - Enhancement - Home/Admin/Catalog/Marketplace/Sharing - Click on app title opens new tab of the app [ID 41524]
+
+From now on, you can open Admin, Catalog, Home, Marketplace, and Sharing in a new tab using middle-click or right-click options. This update makes their titles behave like regular links.
```
Comment (MariekeGO) [inline release-notes/Cloud/DCP_change_log.md:14]:
```suggestion
### 9 December 2024 - Enhancement - Home/Admin/Catalog/Marketplace/Sharing - Right- or middle-clicking app title opens app in new tab [ID 41524]

When you click the title of the Admin, Catalog, Home, Marketplace, or Sharing apps with the right or middle mouse button, this will now open the app again in a new tab.
```

**Review by MariekeGO — APPROVED** [review]
